zigup.zig

```diff
@@ -429,6 +448,44 @@ fn listCompilers(allocator: *Allocator) !void {
     }
 }
 
+fn keepOrUnkeepCompiler(allocator: *Allocator, compiler_name_to_keep_or_unkeep: []const u8, keep_or_unkeep: enum { keep, unkeep }) !void {
+    if (mem.eql(u8, "master", compiler_name_to_keep_or_unkeep)) {
+        std.debug.warn("master is always kept, doing nothing\n", .{});
```

Comment:
This is actually ok.  You can have both a "keep" marker file, and have a compiler be master.